web/styles/popovers.css

```diff
@@ -337,24 +337,42 @@ ul.popover-group-menu-member-list {
     flex-direction: column;
 
     .simplebar-content {
+        display: grid;
+        grid-template-columns:
+            var(--user-group-popover-horizontal-padding)
+            [group-member-icon-start] minmax(5px, auto)
```

Comment:
It might be better to figure out what the widest icon is, and express that maximum values as ems at 16. `auto` makes me a bit hesitant in this context, but it might be okay.

Reply:
The idea of using `grids` with `subgrids` was to make the icons column dynamic, where the largest icon would define the width of that column, and the other smaller icons would center align with respect to that width. So, setting a fixed value defeats that behavior?

Reply:
But understanding your concern with using `auto`, which could sometimes behave unexpectedly.  I have instead used `max-content` as the width of the given grid column which should be the ideal property for the given condition?